### PR TITLE
Minor JavaScript lint fixes.

### PR DESCRIPTION
**Pull request by XhmikosR**
@lgarron: this is sort of untested except for just running it locally without any server.

So, please try it just in case :)

### CODE REVIEW

**Comment by XhmikosR** [comment]
I tested this locally and seems to be fine.

BTW, one thing I noticed is that the list that gets downloaded locally isn't the latest from the live site.

**Comment by lgarron** [comment]
@vcsjones FYI, this will change the syntax but not semantics of the existing JS. TypeScript in the future would still be great. :-D

**Comment by lgarron** [comment]
Thanks! :-)

File: frontend/static/js/form.js [hunk 1]
```diff
@@ -174,65 +176,68 @@ PreloadController.prototype = {
...
   showPreloadEligibility: function(view, domain, issues) {
     switch (worstIssues(issues)) {
       case 'none':
         view.setSummary(
-            'Eligibility: ' + domain +
+          'Eligibility: ' + domain +
             ' is eligible for the HSTS preload list.');
         view.setTheme('theme-green');
         return true;
         break;
```
Comment (XhmikosR) [inline frontend/static/js/form.js]: These are unreachable BTW.

**Pull request merged**